src/sql/workbench/browser/modal/modal.ts

```diff
@@ -2,30 +2,32 @@
  *  Copyright (c) Microsoft Corporation. All rights reserved.
  *  Licensed under the Source EULA. See License.txt in the project root for license information.
  *--------------------------------------------------------------------------------------------*/
-import 'vs/css!./media/modal';
```

Comment:
reorgnized the imports, the only actual change is added import entry for the new method I added.